app/views/smart_proxies/logs/_list.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= render :partial => "smart_proxies/logs/modal" %>
+<div class="fr filter">
+  <%= _('Filter by level:') %> <%= logs_filter_tag %>
+</div>
+<table id="table-proxy-status-logs" class='<%= table_css_classes %> table-fixed'>
+  <thead>
+      <tr>
+          <th class="ca col-md-2 hidden-xs nbsp">Time</th>
+          <th class="ca col-md-1 hidden-sm hidden-xs nbsp">Level</th>
+          <th class="col-md-9 col-sm-10 col-xs-12 nbsp">Message</th>
+      </tr>
+  </thead>
+  <tbody>
+    <% log_entries.each do |log| %>
+      <tr class="<%= logs_color_map[log['level']] || '' %>">
+        <td class="ca col-md-2 hidden-xs nbsp"><%= log['timestamp'] %></td>
+        <td class="ca col-md-1 hidden-sm hidden-xs nbsp">
+          <a href="#"
+            data-toggle="modal"
+            data-target="#backtrace-modal"
+            data-time="<%= log['timestamp'] %>"
+            data-level="<%= log['level'] %>"
+            data-message="<%= log['message'] %>"
+            data-backtrace="<%= log['backtrace'] %>">
+            <%= log['level'] %></td>
```

Comment:
I don't think we should translate log level, we also do not translate log messages in Foreman or Proxy. IMHO this is the threashold when we should stop translating. Someone could be for example grepping logs for additional info. Fixed tag order.